Source: Timo Gurr <tgurr@exherbo.org>
Upstream: No, distribution specific
Reason: Allow specifying an EnvironmentFile

diff -Naur rpcbind-1.2.5/systemd/rpcbind.service.in rpcbind-1.2.5.new/systemd/rpcbind.service.in
--- rpcbind-1.2.5/systemd/rpcbind.service.in	2018-08-15 16:51:19.000000000 +0200
+++ rpcbind-1.2.5.new/systemd/rpcbind.service.in	2018-08-30 14:33:05.145091241 +0200
@@ -11,7 +11,7 @@
 
 [Service]
 Type=notify
-# distro can provide a drop-in adding EnvironmentFile=-/??? if needed.
+EnvironmentFile=-/etc/conf.d/rpcbind.conf
 ExecStart=@_sbindir@/rpcbind $RPCBIND_OPTIONS -w -f
 
 [Install]
